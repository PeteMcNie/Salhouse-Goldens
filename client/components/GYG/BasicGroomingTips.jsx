import React from 'react'

import Image from 'react-bootstrap/Image'
import Container from 'react-bootstrap/Container'
import Row from 'react-bootstrap/Row'
import Col from 'react-bootstrap/Col'

function BasicGroomingTips () {
  return (
    <>
      <div className='mainBody'>
        <Container>
          <Row className='row'>
            <Col lg={6} className='leftCol'>
              <h2 className='bodyTitle gygTitle'>Basic Grooming Tips</h2>
              <p>
                It&apos;s good to learn how to clean their ears out, or at least look in their ears regularly as they can get build
                up of gluggy wax and ear infections. The inside of the ear flap should be pink but not red and hot. If it is then
                it&apos;s possible your Golden has a little yeast infection deeper in the ear and this needs veterinary attention. Sometimes
                they may shake their heads a lot and/or scratch their ear. This again could be the result of an uncomfy feeling in the
                ear that needs attention. Always good to look for these things. If you are confident then you can get ear wipes from the
                pet shop and clean the ears out yourself. Put your finger with the wipe on into the ear and clean around the beginning
                of the ear canal. If it comes up with a lot of brown or black wax, use a clean part of the wipe and keep going until
                you&apos;ve got it cleaned out. Sometimes they can get a lot of ear wax but not actually have an infection. It&apos;s just good to
                keep them clean in these instances.
              </p>
              <p>
                If you find that your Golden&apos;s coat gets too thick with lots of undercoat that you can&apos;t get out and he/she seems
                to be moulting all the time, it can be useful to learn how to strip their coat out yourself. You can use a dog rake as below.
                The rake is an easy implement to use.  Make sure you comb the rake through the neck and the feathers of the front and back
                legs as well as the body of your Golden. You can also use a slicker brush (pictured below) which will get undercoat out and
                can be easier to brush through the feathers of the legs, especially if there are some small knots. Make sure if you are
                pulling on a knot, hold the fur with your other hand halfway down the length to the skin to prevent it from pulling. If
                knots are too big, cut them out gently, being careful to make sure your Golden is still and not likely to move at this time.
              </p>
              <p>
                Nails are another thing that need attention. Walking on pavements doesn&apos;t provide enough trimming for most Goldens&apos;
                nails. Over time they grow longer and longer until they start to touch the ground and turn the toes slightly to one side. You
                can imagine that if you have your toe slightly pushed to the side all the time there will be an ache that gradually becomes
                more uncomfortable and then there is a reluctance to continue to walk much. If the nails are kept trimmed then the bones of
                the toe can just sit in their normal position and your Golden won&apos;t even notice them.
              </p>
            </Col>
            <Col lg={6}>
              <div>
                <Image src='images/gyg2.jpg' className='litterImage' />
                <p className='litterText'>Rake</p>
              </div>
              <div>
                <Image src='images/gyg3.jpg' className='litterImage'/>
                <p className='litterText'>Slicker brush</p>
              </div>
            </Col>
          </Row>
          <hr className='sectionSeparator' id='groomingService' />
        </Container>
      </div>
    </>
  )
}

export default BasicGroomingTips
