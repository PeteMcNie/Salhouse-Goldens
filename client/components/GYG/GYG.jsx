import React from 'react'

function GYG () {
  return (
    <div>
      <p>GYG</p>
    </div>
  )
}

export default GYG
