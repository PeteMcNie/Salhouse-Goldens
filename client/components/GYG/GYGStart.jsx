import React from 'react'

import Container from 'react-bootstrap/Container'
import Row from 'react-bootstrap/Row'
import Col from 'react-bootstrap/Col'
import Image from 'react-bootstrap/Image'

function GYGStart () {
  return (
    <>
      <div className='mainBody'>
        <Container id='gyg'>
          <Row className='row'>
            <Col lg={6} className='leftCol'>
              <h2 className='bodyTitle'>Grooming Your Golden</h2>
              <p>
                Contact Carole on 04 566 0561 or email <a href='mailto:cmcnie@xtra.co.nz?subject=Enquiry regarding Golden Retriever grooming'>cmcnie@xtra.co.nz</a> if you are in the Wellington region (I&apos;m
                actually in Lower Hutt) and would like your Golden groomed.
              </p>
              <p>
                Your Golden Retriever is a beautiful creature at any time of the day or night, but when kept groomed properly they can be
                an even lovelier sight and will feel great. They definitely need some attention to their coats. A brush once or twice a week,
                taking specific care and attention around neck right by the ears, the long coat of the chest and the feathers of the back
                legs and tail. These areas are all particularly prone to getting dirt and little twigs etc in and then form knots that can be
                hard to get out if they are left a long time to embed themselves.
              </p>
            </Col>
            <Col lg={6}>
              <div>
                <Image src='images/gyg13.jpg' fluid className='litterImage'/>
              </div>
            </Col>
          </Row>
          <hr className='sectionSeparator' />
        </Container>
      </div>
    </>
  )
}

export default GYGStart
