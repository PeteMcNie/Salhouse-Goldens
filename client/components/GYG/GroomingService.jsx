import React from 'react'

import Container from 'react-bootstrap/Container'
import Row from 'react-bootstrap/Row'
import Col from 'react-bootstrap/Col'

function GroomingService () {
  return (
    <>
      <div className='mainBody'>
        <Container>
          <Row className='row'>
            <Col className='leftCol'>
              <h2 className='bodyTitle gygTitle' >Grooming Service</h2>
              <p>
                I provide a grooming service for Golden Retrievers. A full groom includes a trim of your Golden&apos;s neck, ears, feet and
                tail and sometimes the length underneath the body if it&apos;s too long. I also remove some of the undercoat and any loose
                fur and knots. I do not shave Golden Retrievers. They have a double coat for a reason, and partly it&apos;s an air
                conditioning unit. The best way to relieve a Golden that you think is feeling the heat is to get the undercoat removed.
                If your dog is shaved, the sun and heat has more access to the body of the dog but you haven&apos;t actually stripped out any
                of the undercoat. All that has been done is to make the thick coat less long instead of actually thinning out the coat
                which will work much better.  If shaved, the undercoat will grow back first and not provide the protection that the coarser
                top-coat does. The dog will feel hotter and the undercoat will get wet when swimming and take a long time to dry which can
                cause skin issues. Most dogs feel the heat in the summer anyway, but with a correct groom there will be the correct
                protection for your dog without ruining the coat.
              </p>
              <p>
                I am not a professionally trained groomer, but I have had years of grooming Goldens for the show ring and love grooming
                this breed.
              </p>
            </Col>
          </Row>
          <hr className='sectionSeparator' id='t&p'/>
        </Container>
      </div>
    </>
  )
}

export default GroomingService
