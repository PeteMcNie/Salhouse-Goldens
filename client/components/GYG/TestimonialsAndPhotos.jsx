import React from 'react'

import Image from 'react-bootstrap/Image'

function TestimonialsAndPhotos () {
  return (
    <>
      <div>
        <h3>Testimonials and Photos</h3>
      </div>
      <div>
        <Image src='images/gyg4.jpg' fluid/>
        <p>Missy after groom:</p>
        <p>
            It was so lovely meeting you and thank you so much for grooming her today! She looks so amazing thanks to you! This is
            probably the best she&apos;s ever looked after a groom so I&apos;m really thankful for your great service. I think she really
            enjoyed it also, and meeting your lovely goldens too! Thank you so much for the photos! It&apos;s evident that she really
            enjoyed being groomed by you. She looks so cute! Definitely having her groomed with you again if you would be happy to
            have her again sometime next year. I really appreciate your wonderful work. - Celina
        </p>
      </div>
      <div>
        <Image src='images/gyg5.jpg' fluid/>
        <Image src='images/gyg6.jpg' fluid/>
        <p>Missy before and after groom</p>
      </div>
      <div>
        <Image src='images/gyg7.jpg' fluid/>
        <Image src='images/gyg8.jpg' fluid/>
        <p>Baxter before and after groom</p>
      </div>
      <div>
        <Image src='images/gyg9.jpg' fluid/>
        <Image src='images/gyg10.jpg' fluid/>
        <p>Billie before and after groom:</p>
        <p>Thank you for grooming Billie you have done a fantastic job and he looks very handsome. I hope he behaved himself. We will be in contact next time we want a groom. - Debbie</p>
      </div>
      <div>
        <Image src='images/gyg11.jpg' fluid/>
        <Image src='images/gyg12.jpg' fluid/>
        <p>Rafa before and after groom</p>
      </div>
      <div>
        <Image src='images/gyg13.jpg' fluid/>
        <p>Rafa front view</p>
      </div>
      <div>
        <Image src='images/gyg14.jpg' fluid/>
        <Image src='images/gyg15.jpg' fluid/>
        <p>Cody before and after groom</p>
      </div>
      <div>
        <Image src='images/gyg16.jpg' fluid/>
        <Image src='images/gyg17.jpg' fluid/>
        <p>Zach before and after groom</p>
      </div>
      <div>
        <Image src='images/gyg18.jpg' fluid/>
        <Image src='images/gyg19.jpg' fluid/>
        <p>Jax before and after groom</p>
      </div>
      <div>
        <Image src='images/gyg20.jpg' fluid/>
        <Image src='images/gyg21.jpg' fluid/>
        <p>Tawnee before and after groom</p>
      </div>
      <div>
        <Image src='images/gyg22.jpg' fluid/>
        <p>Tawnee front view</p>
      </div>
    </>
  )
}

export default TestimonialsAndPhotos
