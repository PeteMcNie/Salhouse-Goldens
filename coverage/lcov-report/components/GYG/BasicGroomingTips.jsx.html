
<!doctype html>
<html lang="en">

<head>
    <title>Code coverage report for components/GYG/BasicGroomingTips.jsx</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../prettify.css" />
    <link rel="stylesheet" href="../../base.css" />
    <link rel="shortcut icon" type="image/x-icon" href="../../favicon.png" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../sort-arrow-sprite.png);
        }
    </style>
</head>
    
<body>
<div class='wrapper'>
    <div class='pad1'>
        <h1><a href="../../index.html">All files</a> / <a href="index.html">components/GYG</a> BasicGroomingTips.jsx</h1>
        <div class='clearfix'>
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Statements</span>
                <span class='fraction'>1/1</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Branches</span>
                <span class='fraction'>0/0</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Functions</span>
                <span class='fraction'>1/1</span>
            </div>
        
            
            <div class='fl pad1y space-right2'>
                <span class="strong">100% </span>
                <span class="quiet">Lines</span>
                <span class='fraction'>1/1</span>
            </div>
        
            
        </div>
        <p class="quiet">
            Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
        </p>
    </div>
    <div class='status-line high'></div>
    <pre><table class="coverage">
<tr><td class="line-count quiet"><a name='L1'></a><a href='#L1'>1</a>
<a name='L2'></a><a href='#L2'>2</a>
<a name='L3'></a><a href='#L3'>3</a>
<a name='L4'></a><a href='#L4'>4</a>
<a name='L5'></a><a href='#L5'>5</a>
<a name='L6'></a><a href='#L6'>6</a>
<a name='L7'></a><a href='#L7'>7</a>
<a name='L8'></a><a href='#L8'>8</a>
<a name='L9'></a><a href='#L9'>9</a>
<a name='L10'></a><a href='#L10'>10</a>
<a name='L11'></a><a href='#L11'>11</a>
<a name='L12'></a><a href='#L12'>12</a>
<a name='L13'></a><a href='#L13'>13</a>
<a name='L14'></a><a href='#L14'>14</a>
<a name='L15'></a><a href='#L15'>15</a>
<a name='L16'></a><a href='#L16'>16</a>
<a name='L17'></a><a href='#L17'>17</a>
<a name='L18'></a><a href='#L18'>18</a>
<a name='L19'></a><a href='#L19'>19</a>
<a name='L20'></a><a href='#L20'>20</a>
<a name='L21'></a><a href='#L21'>21</a>
<a name='L22'></a><a href='#L22'>22</a>
<a name='L23'></a><a href='#L23'>23</a>
<a name='L24'></a><a href='#L24'>24</a>
<a name='L25'></a><a href='#L25'>25</a>
<a name='L26'></a><a href='#L26'>26</a>
<a name='L27'></a><a href='#L27'>27</a>
<a name='L28'></a><a href='#L28'>28</a>
<a name='L29'></a><a href='#L29'>29</a>
<a name='L30'></a><a href='#L30'>30</a>
<a name='L31'></a><a href='#L31'>31</a>
<a name='L32'></a><a href='#L32'>32</a>
<a name='L33'></a><a href='#L33'>33</a>
<a name='L34'></a><a href='#L34'>34</a>
<a name='L35'></a><a href='#L35'>35</a>
<a name='L36'></a><a href='#L36'>36</a>
<a name='L37'></a><a href='#L37'>37</a>
<a name='L38'></a><a href='#L38'>38</a>
<a name='L39'></a><a href='#L39'>39</a>
<a name='L40'></a><a href='#L40'>40</a>
<a name='L41'></a><a href='#L41'>41</a>
<a name='L42'></a><a href='#L42'>42</a>
<a name='L43'></a><a href='#L43'>43</a>
<a name='L44'></a><a href='#L44'>44</a>
<a name='L45'></a><a href='#L45'>45</a>
<a name='L46'></a><a href='#L46'>46</a>
<a name='L47'></a><a href='#L47'>47</a>
<a name='L48'></a><a href='#L48'>48</a>
<a name='L49'></a><a href='#L49'>49</a>
<a name='L50'></a><a href='#L50'>50</a>
<a name='L51'></a><a href='#L51'>51</a>
<a name='L52'></a><a href='#L52'>52</a>
<a name='L53'></a><a href='#L53'>53</a>
<a name='L54'></a><a href='#L54'>54</a></td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React from 'react'
&nbsp;
import Image from 'react-bootstrap/Image'
&nbsp;
function BasicGroomingTips () {
  return (
    &lt;&gt;
      &lt;div&gt;
        &lt;h3 id='basicGroomingTips'&gt;Basic Grooming Tips&lt;/h3&gt;
        &lt;p&gt;
            It&amp;apos;s good to learn how to clean their ears out, or at least look in their ears regularly as they can get build
            up of gluggy wax and ear infections. The inside of the ear flap should be pink but not red and hot. If it is then
            it&amp;apos;s possible your Golden has a little yeast infection deeper in the ear and this needs veterinary attention. Sometimes
            they may shake their heads a lot and/or scratch their ear. This again could be the result of an uncomfy feeling in the
            ear that needs attention. Always good to look for these things. If you are confident then you can get ear wipes from the
            pet shop and clean the ears out yourself. Put your finger with the wipe on into the ear and clean around the beginning
            of the ear canal. If it comes up with a lot of brown or black wax, use a clean part of the wipe and keep going until
            you&amp;apos;ve got it cleaned out. Sometimes they can get a lot of ear wax but not actually have an infection. It&amp;apos;s just good to
            keep them clean in these instances.
        &lt;/p&gt;
        &lt;p&gt;
            If you find that your Golden&amp;apos;s coat gets too thick with lots of undercoat that you can&amp;apos;t get out and he/she seems
            to be moulting all the time, it can be useful to learn how to strip their coat out yourself. You can use a dog rake as below.
            The rake is an easy implement to use.  Make sure you comb the rake through the neck and the feathers of the front and back
            legs as well as the body of your Golden. You can also use a slicker brush (pictured below) which will get undercoat out and
            can be easier to brush through the feathers of the legs, especially if there are some small knots. Make sure if you are
            pulling on a knot, hold the fur with your other hand halfway down the length to the skin to prevent it from pulling. If
            knots are too big, cut them out gently, being careful to make sure your Golden is still and not likely to move at this time.
        &lt;/p&gt;
      &lt;/div&gt;
      &lt;div&gt;
        &lt;Image src='images/gyg2.jpg' /&gt;
        &lt;p&gt;Rake&lt;/p&gt;
      &lt;/div&gt;
      &lt;div&gt;
        &lt;Image src='images/gyg3.jpg' /&gt;
        &lt;p&gt;Slicker brush&lt;/p&gt;
      &lt;/div&gt;
      &lt;div&gt;
        &lt;p&gt;
          Nails are another thing that need attention. Walking on pavements doesn&amp;apos;t provide enough trimming for most Goldens&amp;apos;
          nails. Over time they grow longer and longer until they start to touch the ground and turn the toes slightly to one side. You
          can imagine that if you have your toe slightly pushed to the side all the time there will be an ache that gradually becomes
          more uncomfortable and then there is a reluctance to continue to walk much. If the nails are kept trimmed then the bones of
          the toe can just sit in their normal position and your Golden won&amp;apos;t even notice them.
        &lt;/p&gt;
        &lt;hr /&gt;
      &lt;/div&gt;
    &lt;/&gt;
  )
}
&nbsp;
export default BasicGroomingTips
&nbsp;</pre></td></tr></table></pre>

                <div class='push'></div><!-- for sticky footer -->
            </div><!-- /wrapper -->
            <div class='footer quiet pad2 space-top1 center small'>
                Code coverage generated by
                <a href="https://istanbul.js.org/" target="_blank">istanbul</a>
                at Wed Sep 30 2020 13:24:11 GMT+1300 (New Zealand Daylight Time)
            </div>
        </div>
        <script src="../../prettify.js"></script>
        <script>
            window.onload = function () {
                prettyPrint();
            };
        </script>
        <script src="../../sorter.js"></script>
        <script src="../../block-navigation.js"></script>
    </body>
</html>
    