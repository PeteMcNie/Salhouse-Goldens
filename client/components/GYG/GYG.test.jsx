import React from 'react'
import { render } from '@testing-library/react'

import GYG from './GYG'

test('GYG component matches snapshot', () => {
  const { container } = render(<GYG />)
  expect(container).toMatchSnapshot()
})
