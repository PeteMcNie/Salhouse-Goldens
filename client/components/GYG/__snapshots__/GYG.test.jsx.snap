// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GYG component matches snapshot 1`] = `
<div>
  <nav
    class="navbar navbar-expand-lg navbar-light bg-light fixed-top"
  >
    <a
      class="navbar-brand"
      href="/"
    >
      <img
        alt="Goldens retrievers, phoenix and kula"
        class="shlogo d-inline-block align-top"
        src="images/logo2.png"
      />
    </a>
    <button
      aria-controls="responsive-navbar-nav"
      aria-label="Toggle navigation"
      class="navbar-toggler collapsed"
      type="button"
    >
      <span
        class="navbar-toggler-icon"
      />
    </button>
    <div
      class="navbar-collapse collapse"
      id="responsive-navbar-nav"
    >
      <div
        class="ml-auto navbar-nav"
      >
        <a
          class="nav-link"
          data-rb-event-key="/"
          href="/"
        >
          Home
        </a>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            Our Goldens
          </a>
        </div>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            Grooming Your Golden
          </a>
        </div>
        <a
          class="nav-link"
          data-rb-event-key="/mylifewithanimals"
          href="/mylifewithanimals"
        >
          My Life With Animals
        </a>
        <div
          class="dropdown nav-item"
        >
          <a
            aria-expanded="false"
            aria-haspopup="true"
            class="dropdown-toggle nav-link"
            href="#"
            id="collasible-nav-dropdown"
            role="button"
          >
            About Us
          </a>
        </div>
        <a
          class="nav-link"
          data-rb-event-key="/links"
          href="/links"
        >
          Links
        </a>
        <a
          class="nav-link"
          data-rb-event-key="/contact"
          href="/contact"
        >
          Contact
        </a>
      </div>
    </div>
  </nav>
  <div
    class="jumbotron jumbotron-fluid"
    id="gygj"
  >
    <div>
      <h1
        class="jumbotronTitle GYG"
      >
        Grooming Your Golden
      </h1>
      <p
        class="line"
      />
    </div>
  </div>
  <div
    class="mainBody"
  >
    <div
      class="mainBody"
    >
      <div
        class="container"
      >
        <div
          class="row row"
        >
          <div
            class="leftCol col-lg-6"
          >
            <h2
              class="bodyTitle gygTitle"
            >
              Grooming Your Golden
            </h2>
            <p>
              Contact Carole on 04 566 0561 or email 
              <a
                href="mailto:cmcnie@xtra.co.nz?subject=Enquiry regarding Golden Retriever grooming"
              >
                cmcnie@xtra.co.nz
              </a>
               if you are in the Wellington region (I'm actually in Lower Hutt) and would like your Golden groomed.
            </p>
            <p>
              Your Golden Retriever is a beautiful creature at any time of the day or night, but when kept groomed properly they can be an even lovelier sight and will feel great. They definitely need some attention to their coats. A brush once or twice a week, taking specific care and attention around neck right by the ears, the long coat of the chest and the feathers of the back legs and tail. These areas are all particularly prone to getting dirt and little twigs etc in and then form knots that can be hard to get out if they are left a long time to embed themselves.
            </p>
          </div>
          <div
            class="col-lg-6"
          >
            <div>
              <img
                class="litterImage img-fluid"
                src="images/gyg13.jpg"
              />
            </div>
          </div>
        </div>
        <hr
          class="sectionSeparator"
          id="groomingTips"
        />
      </div>
    </div>
    <div
      class="mainBody"
    >
      <div
        class="container"
      >
        <div
          class="row row"
        >
          <div
            class="leftCol col-lg-6"
          >
            <h2
              class="bodyTitle gygTitle"
            >
              Basic Grooming Tips
            </h2>
            <p>
              It's good to learn how to clean their ears out, or at least look in their ears regularly as they can get build up of gluggy wax and ear infections. The inside of the ear flap should be pink but not red and hot. If it is then it's possible your Golden has a little yeast infection deeper in the ear and this needs veterinary attention. Sometimes they may shake their heads a lot and/or scratch their ear. This again could be the result of an uncomfy feeling in the ear that needs attention. Always good to look for these things. If you are confident then you can get ear wipes from the pet shop and clean the ears out yourself. Put your finger with the wipe on into the ear and clean around the beginning of the ear canal. If it comes up with a lot of brown or black wax, use a clean part of the wipe and keep going until you've got it cleaned out. Sometimes they can get a lot of ear wax but not actually have an infection. It's just good to keep them clean in these instances.
            </p>
            <p>
              If you find that your Golden's coat gets too thick with lots of undercoat that you can't get out and he/she seems to be moulting all the time, it can be useful to learn how to strip their coat out yourself. You can use a dog rake as below. The rake is an easy implement to use.  Make sure you comb the rake through the neck and the feathers of the front and back legs as well as the body of your Golden. You can also use a slicker brush (pictured below) which will get undercoat out and can be easier to brush through the feathers of the legs, especially if there are some small knots. Make sure if you are pulling on a knot, hold the fur with your other hand halfway down the length to the skin to prevent it from pulling. If knots are too big, cut them out gently, being careful to make sure your Golden is still and not likely to move at this time.
            </p>
            <p>
              Nails are another thing that need attention. Walking on pavements doesn't provide enough trimming for most Goldens' nails. Over time they grow longer and longer until they start to touch the ground and turn the toes slightly to one side. You can imagine that if you have your toe slightly pushed to the side all the time there will be an ache that gradually becomes more uncomfortable and then there is a reluctance to continue to walk much. If the nails are kept trimmed then the bones of the toe can just sit in their normal position and your Golden won't even notice them.
            </p>
          </div>
          <div
            class="col-lg-6"
          >
            <div>
              <img
                class="litterImage"
                src="images/gyg2.jpg"
              />
              <p
                class="litterText"
              >
                Rake
              </p>
            </div>
            <div>
              <img
                class="litterImage"
                src="images/gyg3.jpg"
              />
              <p
                class="litterText"
              >
                Slicker brush
              </p>
            </div>
          </div>
        </div>
        <hr
          class="sectionSeparator"
          id="groomingService"
        />
      </div>
    </div>
    <div
      class="mainBody"
    >
      <div
        class="container"
      >
        <div
          class="row row"
        >
          <div
            class="leftCol col"
          >
            <h2
              class="bodyTitle gygTitle"
            >
              Grooming Service
            </h2>
            <p>
              I provide a grooming service for Golden Retrievers. A full groom includes a trim of your Golden's neck, ears, feet and tail and sometimes the length underneath the body if it's too long. I also remove some of the undercoat and any loose fur and knots. I do not shave Golden Retrievers. They have a double coat for a reason, and partly it's an air conditioning unit. The best way to relieve a Golden that you think is feeling the heat is to get the undercoat removed. If your dog is shaved, the sun and heat has more access to the body of the dog but you haven't actually stripped out any of the undercoat. All that has been done is to make the thick coat less long instead of actually thinning out the coat which will work much better.  If shaved, the undercoat will grow back first and not provide the protection that the coarser top coat does. The dog will feel hotter and the undercoat will get wet when swimming and take a long time to dry which can cause skin issues. Most dogs feel the heat in the summer anyway, but with a correct groom there will be the correct protection for your dog without ruining the coat.
            </p>
            <p>
              I am not a professionally trained groomer, but I have had years of grooming Goldens for the show ring and love grooming this breed.
            </p>
          </div>
        </div>
        <hr
          class="sectionSeparator"
          id="t&p"
        />
      </div>
    </div>
    <div
      class="mainBody"
    >
      <div
        class="container"
      >
        <div
          class="row row"
        >
          <div
            class="leftCol col"
          >
            <h2
              class="bodyTitle gygPhotoTitle"
            >
              Testimonials and Photos
            </h2>
          </div>
        </div>
        <div
          class="row"
        >
          <div
            class="leftCol col-lg-6"
          >
            <img
              class="litterImage img-fluid"
              src="images/gyg5.jpg"
            />
            <p
              class="litterText"
            >
              Missy before groom
            </p>
          </div>
          <div
            class="col-lg-6"
          >
            <img
              class="litterImage img-fluid"
              src="images/gyg4.jpg"
            />
            <p
              class="litterText"
            >
              Missy after groom
            </p>
          </div>
        </div>
        <div
          class="row"
        >
          <div
            class="col"
          >
            <p
              class="testimonials"
            >
              It was so lovely meeting you and thank you so much for grooming her today! She looks so amazing thanks to you! This is probably the best she's ever looked after a groom so I'm really thankful for your great service. I think she really enjoyed it also, and meeting your lovely goldens too! Thank you so much for the photos! It's evident that she really enjoyed being groomed by you. She looks so cute! Definitely having her groomed with you again if you would be happy to have her again sometime next year. I really appreciate your wonderful work. - Celina
            </p>
          </div>
        </div>
        <div
          class="row"
        >
          <div
            class="leftCol col-lg-6"
          >
            <img
              class="litterImage img-fluid"
              src="images/gyg7.jpg"
            />
            <p
              class="litterText"
            >
              Baxter before groom
            </p>
          </div>
          <div
            class="col-lg-6"
          >
            <img
              class="litterImage img-fluid"
              src="images/gyg8.jpg"
            />
            <p
              class="litterText"
            >
              Baxter after groom
            </p>
          </div>
        </div>
        <div
          class="row"
        >
          <div
            class="leftCol col-lg-6"
          >
            <img
              class="litterImage img-fluid"
              src="images/gyg9.png"
            />
            <p
              class="litterText"
            >
              Billie before groom
            </p>
          </div>
          <div
            class="col-lg-6"
          >
            <img
              class="litterImage img-fluid"
              src="images/gyg10.png"
            />
            <p
              class="litterText"
            >
              Billie after groom
            </p>
          </div>
        </div>
        <div
          class="row"
        >
          <div
            class="col"
          >
            <p
              class="testimonials"
            >
              Thank you for grooming Billie you have done a fantastic job and he looks very handsome. I hope he behaved himself. We will be in contact next time we want a groom. - Debbie
            </p>
          </div>
        </div>
        <div
          class="row"
        >
          <div
            class="leftCol col-lg-6"
          >
            <img
              class="litterImage img-fluid"
              src="images/gyg11.jpg"
            />
            <p
              class="litterText"
            >
              Rafa before groom
            </p>
          </div>
          <div
            class="col-lg-6"
          >
            <img
              class="litterImage img-fluid"
              src="images/gyg12.jpg"
            />
            <p
              class="litterText"
            >
              Rafa after groom
            </p>
          </div>
        </div>
        <div
          class="row"
        >
          <div
            class="leftCol col-lg-6"
          >
            <img
              class="litterImage img-fluid"
              src="images/gyg13.jpg"
            />
            <p
              class="litterText"
            >
              Rafa front view
            </p>
          </div>
          <div
            class="col-lg-6"
          />
        </div>
        <div
          class="row"
        >
          <div
            class="leftCol col-lg-6"
          >
            <img
              class="litterImage img-fluid"
              src="images/gyg14.jpg"
            />
            <p
              class="litterText"
            >
              Cody before groom
            </p>
          </div>
          <div
            class="col-lg-6"
          >
            <img
              class="litterImage img-fluid"
              src="images/gyg15.jpg"
            />
            <p
              class="litterText"
            >
              Cody after groom
            </p>
          </div>
        </div>
        <div
          class="row"
        >
          <div
            class="leftCol col-lg-6"
          >
            <img
              class="litterImage img-fluid"
              src="images/gyg16.jpg"
            />
            <p
              class="litterText"
            >
              Zach before groom
            </p>
          </div>
          <div
            class="col-lg-6"
          >
            <img
              class="litterImage img-fluid"
              src="images/gyg17.jpg"
            />
            <p
              class="litterText"
            >
              Zach after groom
            </p>
          </div>
        </div>
        <div
          class="row"
        >
          <div
            class="leftCol col-lg-6"
          >
            <img
              class="litterImage img-fluid"
              src="images/gyg18.jpg"
            />
            <p
              class="litterText"
            >
              Jax before groom
            </p>
          </div>
          <div
            class="col-lg-6"
          >
            <img
              class="litterImage img-fluid"
              src="images/gyg19.jpg"
            />
            <p
              class="litterText"
            >
              Jax after groom
            </p>
          </div>
        </div>
        <div
          class="row"
        >
          <div
            class="leftCol col-lg-6"
          >
            <img
              class="litterImage img-fluid"
              src="images/gyg20.jpg"
            />
            <p
              class="litterText"
            >
              Tawnee before groom
            </p>
          </div>
          <div
            class="col-lg-6"
          >
            <img
              class="litterImage img-fluid"
              src="images/gyg21.jpg"
            />
            <p
              class="litterText"
            >
              Tawnee after groom
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
